web_app/dist/main.css

```diff
@@ -0,0 +1,159 @@
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 18px;
+    height: 100%;
+    background: #93ca68;
+    background-image: url(images/bg_logo.png);
+    background-repeat: repeat;   
+}   
+
+#content {
+    background: #fff;
+    color: #545454;
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 60%;
+    height: 90%;
+    margin-top: 50px;
+    padding-bottom: 20px;
+    border-radius: 10px;
+    box-shadow: 5px 25px 20px -10px #606060; 
+}
+
+#sticky-header {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
+
+/* The navigation bar */
+.navbar {
+    display: flex;
+    overflow: hidden;
+    background-color: #333;
+    width: 100%; /* Full width */
+    box-shadow: -5px 20px 50px -10px #656565; 
+}
+
+/* Style the search box inside the navigation bar */
+#search-bar {
+    display: flex;
+    padding: 2px;
+    border: none;
+    margin-top: 10px;
+    margin-right: 16px;
+    margin-left: auto;
+    margin-bottom: 10px;
+    font-size: 17px;
+    border-radius: 5px;
+}
+
+.navbar a {
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+.navbar a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+/* Main content */
+#main {
+    display: flex;
+    flex-direction: column;
+    margin-top: 22px;
+}
+
+.task {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color:honeydew;
+    background-blend-mode: lighten;
+    border-top: 0.2px;
+    border-bottom: 0.2px;
+    border-left: 0px;
+    border-right: 0px;
+    border-style: solid;
+    border-color: darkgray;
+    color: #656565;
+}
+
+.desc {
+    color: #858585;
+    font-size: 14px;
+}
+
+.time {
+    font-size: 30px;
+}
+
+.task-text-spacing {
+    font-size: 25px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 2px;
+    padding-bottom: 2px;
+    margin: 0;
+    
+    border-top: 0px;
+    border-bottom: 0px;
+    border-left: 0px;
+    border-right: 1.5px;
+    border-style: solid;
+    border-color: darkgray; 
+}
+
+.task-description-container {
```

Comment:
I say move this above `task-description`, since this contains that element. I'm also not super sold on the name (since it contains both the name and the description), but can't think of a better one at the moment